templates/base.html

```diff
@@ -121,11 +54,34 @@
     <div class="u-hide" id="contact-form-container" data-form-location="/get-in-touch" data-form-id="1337" data-lp-id="2313" data-return-url="http://juju.is/thank-you" data-lp-url=""></div>
     <script src="{{ versioned_static('js/dynamic-contact-form.js') }}"></script>
 
-    <script src="/static/js/build/global-nav/global-nav.js"></script>
+    <script src="/static/js/navigation.js"></script>
     <script src="/static/js/build/cookie-policy/cookie-policy.js"></script>
     <script>
-      canonicalGlobalNav.createNav({ maxWidth: "72rem", showLogins: false , hiring: 'https://juju.is/careers' });
-      cpNs.cookiePolicy();
+      document.addEventListener("DOMContentLoaded", () => {
+        const selector = "[data-js='sticky-nav-observer']";
+        const observerEl = document.querySelector(selector);
+        const nav = document.getElementById("navigation");
+
+        if (observerEl && nav) {
+          let observer = new IntersectionObserver(
+            function (entries) {
+              // no intersection with screen
+              if (entries[0].intersectionRatio === 0) {
+                nav.classList.add("is-sticky");
+                // fully intersects with screen
```

Comment:
These comments are not clear to someone that does not understand the `IntersectionObserver` API. Can you try and explain it in plainer language.